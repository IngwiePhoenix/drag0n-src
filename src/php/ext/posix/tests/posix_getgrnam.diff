001+ array(4) {
002+   ["name"]=>
003+   string(0) ""
004+   ["passwd"]=>
005+   string(0) ""
006+   ["members"]=>
007+   array(0) {
008+   }
009+   ["gid"]=>
010+   int(0)
011+ }
001- bool(false)
003- bool(false)
013+ array(4) {
014+   ["name"]=>
015+   string(0) ""
016+   ["passwd"]=>
017+   string(0) ""
018+   ["members"]=>
019+   array(0) {
020+   }
021+   ["gid"]=>
022+   int(0)
023+ }