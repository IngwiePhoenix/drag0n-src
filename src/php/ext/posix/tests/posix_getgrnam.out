array(4) {
  ["name"]=>
  string(0) ""
  ["passwd"]=>
  string(0) ""
  ["members"]=>
  array(0) {
  }
  ["gid"]=>
  int(0)
}
bool(false)
array(4) {
  ["name"]=>
  string(0) ""
  ["passwd"]=>
  string(0) ""
  ["members"]=>
  array(0) {
  }
  ["gid"]=>
  int(0)
}